src/workload.rs

```diff
@@ -531,6 +555,11 @@ impl WorkloadStore {
                     .insert((service_sock_addr, port.target_port as u16));
             }
         }
+        if let Some(tx) = self.cert_tx.as_mut() {
+            if let Err(e) = tx.try_send(widentity) {
```

Comment:
future improvement would be to store a list of identities we already warmed and do not re-warm them. Re-warm is fairly cheap (we have a cache), but not as cheap as a set lookup.